menas/ui/service/RuleService.js

```diff
@@ -51,30 +51,12 @@ var RuleService = new function () {
     })
   };
 
-  this.disableRule = function (sId, iVersion) {
-    var uri = "api/rule/disable/" + encodeURI(sId);
-    if (typeof (iVersion) !== "undefined") {
-      uri += "/" + encodeURI(iVersion)
-    }
+  this.removeRule = function (oCurrentDataset, iRuleIndex) {
+    let conformance = oCurrentDataset["conformance"].filter((_, index) => index !== iRuleIndex);
+    let newDataset = oCurrentDataset;
```

Comment:
Question: Why the new variable declaration? I don't think the structure is copied, or is it?

Reply:
That is true, I did alias of that variable. It is bit confusing so I've changed it.